README.md

```diff
@@ -10,9 +10,9 @@ Finished certifications is checked:
 
 ## Reactjs.org
 
-Tic-tac-toe tutorial game is built and its source code [./my-app/](TODO:Add relative path link from the repository). Implementation goals:
- - [ ] With class components
- - [ ] With functional components + hooks
+Tic-tac-toe tutorial game is built and its source code [source kod](https://github.com/HumeyraSezgin/innovance-frontend-internship/tree/main/my-app). Implementation goals:
```

Comment:
Right now, people see this text in your README.md: `Tic-tac-toe tutorial game is built and its source code source kod`. Are you OK with that? Looks like something is not right.

Plus, giving relative path would be better. (`./my-app`). What happens when I fork your repo, or rename it? Due to url change, current url would be wrong.